src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/App.razor

```diff
@@ -1,11 +1,18 @@
-﻿<PageTitle>To Do app!</PageTitle>
-<Router AppAssembly="@GetType().Assembly">
-	<Found Context="routeData">
-		<RouteView RouteData="@routeData"  DefaultLayout="@typeof(MainLayout)" />
-	</Found>
-	<NotFound>
-		<LayoutView>
-			<p>Sorry, there's nothing at this address.</p>
-		</LayoutView>
-	</NotFound>
-</Router>
+﻿<CascadingAuthenticationState>
+    <PageTitle>To Do app!</PageTitle>
+    <Router AppAssembly="@GetType().Assembly">
+        <Found Context="routeData">
+            <AuthorizeRouteView RouteData="@routeData" DefaultLayout="@typeof(MainLayout)">
+                <NotAuthorized>
+                    <RedirectToSignIn />
```

Comment:
Why not using sign-in component directly here?

Reply:
It's a page, not a simple component, using pages as components is a bad practice.

Reply:
but we need to handle the `redirectUrl` in this solution which will be handled in a separate issue.